frontend/src/lib/templates/singleCardPage/SingleCardPage.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import {Page} from '../page';
+  import type {SingleCardPageProps} from './SingleCardPage.type';
+
+  type $$Props = SingleCardPageProps;
+
+  export let cardClass: $$Props['cardClass'] = '';
+
+  // Merge restProps classes
+  $$restProps.class = `bg-base-300 ${$$restProps.class ?? ''}`;
```

Comment:
The base color is not needed, as the cards added on top of this template will most likely be white. I guess you want to use this template just to define some margins. am I wrong?

Reply:
The base color here sets the page backround to teal so that cards with white background stand out from it. In addition to this, the template sets a proper width for the card and does some little layout tweaks.